packages/components/src/components/Dialog/Dialog.tsx

```diff
@@ -233,14 +253,20 @@ export const DialogHeader = (
 DialogHeader.displayName = 'Dialog.Header';
 
 export const DialogFooter = (
-    { showBorder = true, children, 'data-test-id': dataTestId = 'fondue-dialog-footer' }: DialogFooterProps,
+    {
+        padding = 'compact',
```

Comment:
if we set a default value here, wouldn't it will always override the dialog-wide padding